autotest/units/001_one_port/076_rfc_5549_route/gen.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from scapy.all import *
+from scapy.contrib.mpls import MPLS
+
+
+def write_pcap(filename, *packetsList):
+	if len(packetsList) == 0:
+		PcapWriter(filename)._write_header(Ether())
+		return
+
+	PcapWriter(filename)
+
+	for packets in packetsList:
+		if type(packets) == list:
+			for packet in packets:
+				packet.time = 0
+				wrpcap(filename, [p for p in packet], append=True)
+		else:
+			packets.time = 0
+			wrpcap(filename, [p for p in packets], append=True)
```

Comment:
It's better to use PcapWriter's method `write` instead of opening file each time:
https://github.com/yanet-platform/yanet/pull/197#discussion_r1608293514